examples/lighting-app/mtk/README.md

```diff
@@ -0,0 +1,256 @@
+#CHIP MT793X Lighting Example
+
+An example showing the use of CHIP on the MediaTek MT793X.
+
+<hr>
+
+-   [CHIP MT793X Lighting Example](#chip-mt793x-lighting-example)
+    -   [Introduction](#introduction)
+    -   [Building](#building)
+        -   [Note](#note)
+    -   [Flashing the Application](#flashing-the-application)
+    -   [Viewing Logging Output](#viewing-logging-output)
+    -   [Running the Complete Example](#running-the-complete-example)
+        -   [Notes](#notes)
+    -   [Running RPC console](#running-rpc-console)
+
+<hr>
+
+<a name="intro"></a>
+
+## Introduction
+
+The MT793X lighting example provides a baseline demonstration of a Light control
+device, built using CHIP and the MediaTek Filogic SDK. It can be controlled by a
+Chip controller over Wi-Fi network..
+
+The MT793X device can be commissioned over Bluetooth Low Energy where the device
+and the Chip controller will exchange security information with the Rendez-vous
+procedure. Network credentials are then provided to the MT793X device which will
+then join the network.
+
+The lighting example is intended to serve both as a means to explore the
+workings of CHIP as well as a template for creating real products based on the
+Silicon Labs platform.
+
+<a name="building"></a>
+
+## Building
+
+-   Download the
+    [Simplicity Commander](https://www.silabs.com/mcu/programming-options)
+    command line tool, and ensure that `commander` is your shell search path.
+    (For Mac OS X, `commander` is located inside
+    `Commander.app/Contents/MacOS/`.)
+
+-   Download and install a suitable ARM gcc tool chain:
+    [GNU Arm Embedded Toolchain 9-2019-q4-major](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads)
+
+-   Install some additional tools(likely already present for CHIP developers):
+
+#Linux \$ sudo apt-get install git libwebkitgtk-1.0-0 ninja-build
+
+#Mac OS X \$ brew install ninja
+
+-   Supported hardware:
+
+*   Build the example application:
+
+
+    Use GN/Ninja directly
+
+          $ cd ~/connectedhomeip/examples/lighting-app/mtk
+          $ git submodule update --init
+          $ source third_party/connectedhomeip/scripts/activate.sh
+          $ gn gen out/debug
```

Comment:
please add support in `scripts/build/build_examples.py` as well. gn builds should be tivial (there is already a gn base class there that you can use).